daemon/execdriver/native/template/default_template_linux.go

```diff
@@ -94,5 +97,10 @@ func New() *configs.Config {
 		container.AppArmorProfile = "docker-default"
 	}
 
+	if SystemdCgroups {
+		container.Cgroups.Parent = "system.slice"
```

Comment:
runc could be used with systemd cgroups support (once proper support is added) without docker integration so we could handle that separately.